README.md

```diff
@@ -26,45 +29,37 @@ The goal of this project is the easy creation of W3C Thing Description instances
 - Editing the Thing Description / Thing Model
 - Validating the Thing Description / ThingModel
 - Exporting the Thing Description / ThingModel from the visual representation into JSON-LD
+- Reading/writing/observing exposed properties value exposed by a proxy
 
-## Technologies
 
+## Technologies
 - React
 - TailwindCSS
 
 ## Contribution guide
-
-Please follow our [contribution guide](./CONTRIBUTING.md).
+Any contribution to this project is welcome. If you want to report a bug or have a question, please check the [issue list](https://github.com/eclipse/editdor/issues) or create a new issue. Many thanks. 
 
 ## License
```

Comment:
not part of this PR but actually the license is eclipse or W3C. the code headers have the correct mention